.github/workflows/check-dist.yml

```diff
@@ -21,12 +21,12 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v4
 
-      - name: Set Node.js 16.x
-        uses: actions/setup-node@v3.7.0
+      - name: Set Node.js 18
+        uses: actions/setup-node@v4.0.0
         with:
-          node-version: 16.x
+          node-version: 18
```

Comment:
it reads 20 in the readme below:
https://github.com/Azure/setup-azd/pull/87/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R59